components/mpas-ocean/src/Registry.xml

```diff
@@ -655,6 +659,20 @@
 					description="Option of Langmuir enhanced entrainment parameterization"
 					possible_values="NONE, LWF16, LF17, RWHGK16"
 		/>
+		<nml_option name="config_cvmix_kpp_use_active_wave" type="logical" default_value=".false." units="NA"
+					description="Flag for Langmuir enchancement factor using prognostic waves"
+					possible_values=""
+		/>
+	</nml_record>
+	<nml_record name="wave_coupling" mode="forward">
+		<nml_option name="config_use_active_wave" type="logical" default_value=".false." units="NA"
+					description="Flag for using prognostic waves"
+					possible_values=".true. or .false."
+		/>
+		<nml_option name="config_n_stokes_drift_wavenumber_partitions" type="integer" default_value="6" units="NA"
```

Comment:
Is there an option to turn stokes drift on or off? Or is that in the `config_use_active_wave`? If so, please add that to the description.

Reply:
`config_use_active_wave` turns off/on all wave related fields and  the stokes drift profile calculation. `config_cvmix_kpp_use_active_wave` specifically turns on/off the active wave Langmuir parameterization.

Reply:
I should probably add a condition that ensures `config_cvmix_kpp_use_active_wave` can't be `.true.` if `config_use_active_wave ` is `.false.`

Reply:
Yes, good idea. OK, I think the flag name `config_use_active_wave` is fine if you add more details to the description (i.e. turns on all wave related fields and the stokes drift profile calculation) and also say that it requires to be run in E3SM with WW3 on for active_wave to work.

Reply:
Great, I'll be sure to do that.

Reply:
I tried to be more descriptive in the latest commit, let me know what you think.